frontend/templates/views/partials/component-headline.j2

```diff
@@ -12,3 +12,13 @@
 </div>
 {% endif %}
 </h1>
+
+{% do doc.styles.addCssFile('css/components/atoms/pill.css') %}
+<div class="ap--flags">
+{% if doc.experimental %}
+  <div class="ap-a-pill ap-a-pill-small ap-a-pill-meta">{{ _('Experimental') }}</div>
+{% endif %}
+{% if doc.deprecated %}
+  <div class="ap-a-pill ap-a-pill-small ap-a-pill-meta">{{ _('Deprecated') }}</div>
+{% endif %}
+</div>
```

Comment:
Guard this with `{% if doc.experimental or doc.deprecated %}` as most of the components do not need this CSS.